ansible/playbooks/30-discourse.yml

```diff
@@ -0,0 +1,22 @@
+---
+- hosts: tachikoma
+  become: yes
+  tasks:
+    - name: Install Docker
+      apt:
+        name: docker.io
+        state: present
+    - name: Clone discourse git repo
+      git:
+        repo: https://github.com/discourse/discourse_docker.git
+        dest: /var/discourse
+    - name: Copy discourse configuration
+      copy:
+        src: discourse/app.yml
```

Comment:
this file isn't included so it won't work

Reply:
file is now included